Misc/NEWS.d/next/Library/2025-07-04-23-45-00.gh-issue-136306.O1YLIU.rst

```diff
@@ -0,0 +1 @@
+:mod:`ssl` can now get and set groups used for key agreement.
```

Comment:
We need a what's new entry as well.

Reply:
I haven't done a what's new entry before - where would I add this? Is it part of the NEWS entry or something separate?

Reply:
It's something in Doc/whatsnew/3.15.rst. See the other entries and add an entry for the ssl module under features section (if the ssl section doesn't exist, create one).

Reply:
Ok - I've added a whatsnew entry under the ssl section of "Improved modules".

Let me know if you think there's too much detail there.